tsl/src/compression/create.c

```diff
@@ -254,7 +260,7 @@ build_columndefs(CompressionSettings *settings, Oid src_relid)
 		 * respective compressed column, because they are accessed before
 		 * decompression.
 		 */
-		bool is_orderby = ts_array_is_member(settings->fd.orderby, NameStr(attr->attname));
+		const bool is_orderby = ts_array_is_member(settings->fd.orderby, NameStr(attr->attname));
```

Comment:
Seems like we can't use bloom indexing on orderby columns. I can see that being actually useful if you are dealing with high cardinality ids which means you can't use it as a segmentby column.

Is there a good reason for this?

Reply:
For orderby, minmax should be better, because you sort the close values together. At least for the leading column. We can also have trailing orderby columns that have groups of only a few values, but are still useful for better compression. In this case minmax for the entire batch won't do much. I'm not sure, I think we just have to make it configurable, and this heuristic is good enough for the time being.